schema/data-actionsauth.json

```diff
@@ -81,10 +81,8 @@
     ["NIPSWeb", "live", "AUTH_USENWLIVE"],
     ["NIPSWeb", "load_auto_managed", "AUTH_USENIPSWEB"],
     ["NIPSWeb", "load_aux_lib", "AUTH_USENIPSWEB"],
-    ["NIPSWeb", "managed_play", "AUTH_USENIPSWEB"],
     ["NIPSWeb", "manage_library", "AUTH_USENIPSWEB"],
     ["NIPSWeb", "recv_ops", "AUTH_USENIPSWEB"],
-    ["NIPSWeb", "secure_play", "AUTH_USENIPSWEB"],
```

Comment:
Shouldn't this need a `GLOBAL_ACCESS` to replace it? I'd imagine this hits the GUI MyRadio login check code

Reply:
Woops, yes.